2024-11-23 12:30:44 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:32:39 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 22776 /T /F
ERROR: The process "22776" not found.

2024-11-23 12:33:44 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:34:13 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:36:02 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 19888 /T /F
ERROR: The process "19888" not found.

2024-11-23 12:36:08 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:43:38 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:52:03 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:56:46 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 18548 /T /F
ERROR: The process "18548" not found.

2024-11-23 12:58:05 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 12:58:59 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 13972 /T /F
ERROR: The process "13972" not found.

2024-11-23 13:03:23 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 13:13:52 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 13:30:06 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 13:34:27 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 15364 /T /F
ERROR: The process "15364" not found.

2024-11-23 13:40:53 [user-service] error: Receive Image Server stderr: file:///C:/Apps/LazyEnder/backend/receiveImage.js:27
    const wss = new WebSocket.Server({ port: 8080 });
                ^

TypeError: WebSocket.Server is not a constructor
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:27:17)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:105:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)

Node.js v22.11.0

2024-11-23 13:41:53 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 6972 /T /F
ERROR: The process "6972" not found.

2024-11-23 13:41:56 [user-service] error: Backend Server stderr: node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:27:17)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:105:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v22.11.0

2024-11-23 13:43:38 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 9912 /T /F
ERROR: The process "9912" not found.

2024-11-23 13:43:44 [user-service] error: Backend Server stderr: node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8081
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:27:17)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:105:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8081
}

Node.js v22.11.0

2024-11-23 13:44:13 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 14444 /T /F
ERROR: The process "14444" not found.

2024-11-23 13:44:38 [user-service] error: Backend Server stderr: node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8081
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:27:17)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:105:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8081
}

Node.js v22.11.0

2024-11-23 13:46:30 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 9088 /T /F
ERROR: The process "9088" not found.

2024-11-23 13:46:33 [user-service] error: Backend Server stderr: node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8082
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:27:17)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:105:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8082
}

Node.js v22.11.0

2024-11-23 13:49:34 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 14596 /T /F
ERROR: The process "14596" not found.

2024-11-23 13:49:38 [user-service] error: Backend Server stderr: node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8082
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:28:17)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:106:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8082
}

Node.js v22.11.0

2024-11-23 13:51:49 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 10020 /T /F
ERROR: The process "10020" not found.

2024-11-23 13:51:54 [user-service] error: Error starting the server: listen EADDRINUSE: address already in use 25.39.138.224:5001
2024-11-23 13:53:10 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 1396 /T /F
ERROR: The process "1396" not found.

2024-11-23 13:55:12 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 320 /T /F
ERROR: The process "320" not found.

2024-11-23 13:57:42 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 6304 /T /F
ERROR: The process "6304" not found.

2024-11-23 14:04:38 [user-service] error: Error starting the server: Cannot access 'server' before initialization
2024-11-23 14:04:38 [user-service] error: Error starting the server: Cannot access 'server' before initialization
2024-11-23 14:04:50 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 13428 /T /F
ERROR: The process "13428" not found.

2024-11-23 14:37:13 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 17620 /T /F
ERROR: The process "17620" not found.

2024-11-23 14:43:09 [user-service] error: Error stopping undefined: Command failed: taskkill /pid 21232 /T /F
ERROR: The process "21232" not found.

2024-11-23 14:45:53 [user-service] error: Error executing findFreePort.js: Error: Command failed: node C:\Apps\LazyEnder\backend\findFreePort.js
file:///C:/Apps/LazyEnder/backend/findFreePort.js:6
const configPath = path.join(__dirname, 'config', 'websocket.js');
                             ^

ReferenceError: __dirname is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and 'C:\Apps\LazyEnder\package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at file:///C:/Apps/LazyEnder/backend/findFreePort.js:6:30
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)

Node.js v22.11.0

2024-11-23 14:46:52 [user-service] error: Error starting Backend Server: Error: Command failed: node C:\Apps\LazyEnder\backend\server.js
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:30:21)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:108:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v22.11.0

2024-11-23 14:49:04 [user-service] error: Error starting Backend Server: Error: Command failed: node C:\Apps\LazyEnder\backend\server.js
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:30:21)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:108:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v22.11.0

2024-11-23 14:49:59 [user-service] error: Error starting Backend Server: Error: Command failed: node C:\Apps\LazyEnder\backend\server.js
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:30:21)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:108:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v22.11.0

2024-11-23 14:54:04 [user-service] error: Error starting Backend Server: Error: Command failed: node C:\Apps\LazyEnder\backend\server.js
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:30:21)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:108:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v22.11.0

2024-11-23 14:54:30 [user-service] error: Error starting Backend Server: Error: Command failed: node C:\Apps\LazyEnder\backend\server.js
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1907:16)
    at listenInCluster (node:net:1964:12)
    at Server.listen (node:net:2066:7)
    at new WebSocketServer (C:\Apps\LazyEnder\node_modules\ws\lib\websocket-server.js:102:20)
    at startServer (file:///C:/Apps/LazyEnder/backend/receiveImage.js:30:21)
    at file:///C:/Apps/LazyEnder/backend/receiveImage.js:108:1
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:518:28)
    at emitErrorNT (node:net:1943:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v22.11.0

